atest/resources/TestCheckerLibrary.py

```diff
@@ -14,48 +14,61 @@
 from robot.result.model import Body, Iterations
 from robot.utils.asserts import assert_equal
 
-
-class NoSlotsKeyword(Keyword):
+class BodyGetter:
```

Comment:
`BodyGetter` is an ok name, but perhaps something like `WithDirectBodyAccess` could be a bit better. We've used `With` prefix also elsewhere with mixins and it looks good also when used like `NoSlotsFor(For, WithDirecBodyAccess)`.

Related to the above, a better name for `NoSlotsWhatever` would be good too. That `NoSlotsPrefix has never been that good, but now when these objects have also other functionality than not having `__slots__` it's even worse. Would an `Atest` prefix like `AtestFor` be fine?